content/controller/admin-guides/backup-restore/backup-restore-cluster-config.md

```diff
@@ -19,7 +19,7 @@ After installing F5 NGINX Controller, you should back up the cluster config and
   /opt/nginx-controller/helper.sh cluster-config save
   ```
 
-  The file is saved to `/opt/nginx-controller/cluster-config.tgz`.
+  The file is saved to `/opt/nginx-controller/cluster-config.tar.gz`.
```

Comment:
This is not correct.

We have changed our guidelines, but the behavior of Controller has not changed. The product is not being updated so this change will probably never happen.

There are other instances of this same situation in this PR.

Reply:
@Dindihub , this was based on my mistake. 

Functionally, tar.gz and tgz extensions are identical. However, that doesn't change how the file might appear, say from a download. My mistake was to not specify those exceptions.

We appreciate your contribution. I'm going to see if I can exclude those exceptions in this PR (I'll apply them directly). Final approval may take a few days, as some of us are either on PTO or in conferences today and next week.

Reply:
> @Dindihub , this was based on my mistake.
> 
> Functionally, tar.gz and tgz extensions are identical. However, that doesn't change how the file might appear, say from a download. My mistake was to not specify those exceptions.
> 
> We appreciate your contribution. I'm going to see if I can exclude those exceptions in this PR (I'll apply them directly). Final approval may take a few days, as some of us are either on PTO or in conferences today and next week.

Thanks for the clarification @mjang. I look forward to your review.